weapons.yaml

```diff
@@ -771,29 +675,13 @@ FlakTrackAAGun:
 		Explosions: flak_puff_AA
 
 FlakWeapon:
+	Inherits: FlakTrackAAGun
 	ReloadDelay: 20
 	Range: 12c0
 	Report: bflaatta.wav, bflaattb.wav, bflaattc.wav, bflaattd.wav
-	ValidTargets: Air
-	Projectile: Bullet
-		Speed: 100c0
 	Warhead@1Dam: SpreadDamage
-		Spread: 120
 		Damage: 40
-		ValidTargets: Air
-		Versus:
-			None: 150
-			Flak: 80
-			Plate: 50
-			Light: 100
-			Medium: 20
-			Heavy: 20
-			Wood: 0
-			Steel: 0
-			Concrete: 0
 		DamageTypes: BulletDeath
```

Comment:
Oh, right. My derp. ^^ We might want to remove the "TriggerProne" types then from `FlakTrackAAGun`.